modules/troubleshooting-common-problems-in-jupyter-for-administrators.adoc

```diff
@@ -123,6 +123,10 @@ endif::[]
 
 * Work with the user to identify files that can be deleted from the `/opt/app-root/src` directory on their notebook server to free up their existing storage space.
 
+[NOTE]
+--
+When you delete files using the JupyterLab file explorer, the files move to your local trash folder. To free up storage space for notebooks, you must permanently delete these files.
```

Comment:
By "local", we mean the local trash folder on your computer, correct? If so, I would say that explicitly: "the trash folder on your computer".

Otherwise, this looks good to me.